stable/vmware-exporter/templates/secret.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
It would be a good practive allowing using an existing secret.

Reply:
Yes :) 
I would like to add that additionally in another PR if possible.